kotsadm/kustomize/overlays/dev/kustomization.yaml

```diff
@@ -1,6 +1,6 @@
 bases:
   - ../../base
-  - ./minio
+  # - ./minio
```

Comment:
i don't see where registry is added to replace minio in dev

Reply:
8 lines down